sdk/search/search-documents/README.md

```diff
@@ -782,6 +808,29 @@ async function main() {
 main();
 ```
 
+#### Create a DataSource
+```js
+const { SearchServiceClient, AzureKeyCredential } = require("@azure/search-documents");
+
+const client = new SearchServiceClient("<endpoint>", new AzureKeyCredential("<apiKey>"));
+
+async function main() {
+  const dataSource = await client.createDataSource({
+    name: 'my-data-source-2',
+    description: 'My Data Source 1',
+    type: 'cosmosdb',
+    container: {
+      name: 'my-container-1'
+    },    
+    credentials: {
+      connectionString: 'AccountEndpoint=https://hotels-docbb.documents.azure.com:443/;AccountKey=4UPsNZyFAjgZ1tzHPGZaxS09XcwLrIawbXBWk6IixcxJoSePTcjBn0mi53XiKWu8MaUgowUhIovOv7kjksqAug==;Database=SampleData'
```

Comment:
maybe put a comment to the samples page where this connection string came from in case it gets updated?

Reply:
Done. Will update in the next commit